source/css/_common/components/post/post-title.styl

```diff
@@ -48,7 +48,6 @@
   }
 
   .fa {
-    font-size: 20px;
```

Comment:
Screens: before and after?

Reply:
No changes. Because of `.fa {font-size: 1em}`

Reply:
Before:
![image](https://user-images.githubusercontent.com/16944225/54815173-a55fd200-4c91-11e9-9e55-c9937491a4d0.png)

After:
![image](https://user-images.githubusercontent.com/16944225/54815124-8a8d5d80-4c91-11e9-8940-6f28a0a3f9bb.png)

Exturl icon too big.